### ISSUE TITLE
Minor classname correction to List README

### ISSUE THREAD

**Issue by robertkern** [material-components/material-components-web#481]
The Two-line lists section has some sample html. The secondary text line is currently using the class `mdc-list-item__text__primary` but should probably use the class `mdc-list-item__text__secondary`

https://github.com/material-components/material-components-web/tree/master/packages/mdc-list#two-line-lists

Currently:

```html
<ul class="mdc-list mdc-list--two-line">
  <li class="mdc-list-item">
    <span class="mdc-list-item__text">
      <span class="mdc-list-item__text__primary">Two-line item</span>
      <span class="mdc-list-item__text__primary">Secondary text</span>
    </span>
  </li>
</ul>
```

Should be:

```html
<ul class="mdc-list mdc-list--two-line">
  <li class="mdc-list-item">
    <span class="mdc-list-item__text">
      <span class="mdc-list-item__text__primary">Two-line item</span>
      <span class="mdc-list-item__text__secondary">Secondary text</span>
    </span>
  </li>
</ul>
```

**Comment by traviskaufman** [comment]
Thanks for reporting! This is being addressed by #439

**Comment by robertkern** [comment]
Awesome that simplifies things a bit!
